src/main/resources/admin/tools/content-type-manager/components/component-file-handler.html

```diff
@@ -31,6 +31,62 @@
         };
       },
       methods: {
+        newCty(){
+          if(this.checkBrowserCompatibility()){
+            this.fileHandle = null;
+            this.$store.commit("resetJson");
+            this.$store.commit("setFileName", { data: ' ' });
+          }
+        },
+        async loadCty() {
```

Comment:
My suggestion to improve the loadCty. (Feel free to ignore) Its pseudo code so you think to fill in the gaps and create the function that don't exist.
```
async loadCty function () {
    const file = await selectFile();
    await showLoadingIcon()
    await setJson(file.json())
    await removeLoadingIcon()
}
```
Don't know if its possible to create this behavior.

Reply:
I tried to improve the _loadCty_ function following some of your suggestions. Please take a look at the last commit :smile: